### ISSUE TITLE
strokeWidth 0 does not work on Text and IText

### ISSUE THREAD

**Issue by riaan53** [kangax/fabric.js#1905]
Hi All,

A strokeWidth of 1 is added to Text or IText when setting strokeWidth to 0 with a stroke color defined to something other than the text color. 

When defining a stroke on shape objects and setting strokeWidth to 0 the object does not display a stroke - do you expect the same behaviour for Text and IText?

See http://jsfiddle.net/0s6z8k5g/1/ 
 
Thank you!

Regards,
Riaan

**Comment by asturur** [comment]
The problem is here:

```
if ((!this.stroke || this.strokeWidth === 0) && !this._skipFillStrokeCheck) {
   return
}
```

In iText class the _skipFillStrokeCheck start with true, so even if there is no strokeWidth, you do not return.

Not really a bug, maybe some old feature or some strange thing that if @kangax does not stop me, i'm gonna change with #1872 to false.

**Comment by riaan53** [comment]
Thank you again! :)

**Comment by asturur** [comment]
Closed via #1872.
